templates/package.html

```diff
@@ -2,7 +2,6 @@
 <html>
     <head>
         <meta name="go-import" content="{{ .CanonicalURL }} git https://{{ .Repo }}">
-        <meta name="go-source" content="{{ .CanonicalURL }} https://{{ .Repo }} https://{{ .Repo }}/tree/{{ .Branch }}{/dir} https://{{ .Repo }}/tree/{{ .Branch }}{/dir}/{file}#L{line}">
```

Comment:
FYI, with this removed, we also don't need the Branch field.
We can drop the Branch field from the template data passed in at line 131 in handler.go.
(We can also drop the corresponding config field, but that can be deferred.)